docs/pages/en-us/Kibana-Reporting.md

```diff
@@ -0,0 +1,104 @@
+---
+layout: default
+title: Reporting
+
+banner:
+    icon: 'assets/img/auditing-graphic.png'
+    title: Auditing
+    subtitle: Traceability, accountability, analytics, archiving and reporting of application tenant events.
+    links:
+        - title: GitHub
+          url: https://github.com/CAFAudit/audit-service
+---
+
+# Kibana Reporting
+
+[Kibana](https://www.elastic.co/products/kibana/) provides reporting capability for audit events data indexed into Elasticsearch. It is a browser based analytics and visualization platform  which can be used to search, view, and interact with the audit data stored in Elasticsearch indices. Kibana provides the ability to perform data analysis and visualize this audit data in a variety of charts, tables, and maps.
+
+## Getting Started
+
+You need to perform the following steps to get started with Kibana reporting.
+
+1. Use the [Audit Service Deploy](https://github.com/CAFAudit/audit-service-deploy) Project to deploy the Elasticsearch cluster and Kibana.
+2. Utilise the [Audit Monkey](https://github.com/CAFAudit/audit-service/tree/develop/caf-audit-monkey) to send audit event messages to Elasticsearch.
+3. Define an index pattern in Kibana. 
+4. Import the set of [saved dashboards, saved searches and visualizations](https://github.com/CAFAudit/audit-service-deploy/blob/develop/kibana/saved-objects.json) into Kibana which can be used to explore and visualize the audit data in several ways.
```

Comment:
Yes I fully expected this. The reason being CAF-2686 is awaiting review. This link will only work when all changes for CAF-2686 have been committed.